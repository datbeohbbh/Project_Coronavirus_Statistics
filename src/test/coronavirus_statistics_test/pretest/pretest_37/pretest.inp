50 100
q 1 37 415
q 1 31 239
q 1 44 148
? 4 1
q 2 37 83
q 1 6 57
q 1 47 108
? 4 2
? 4 3
q 2 37 87
q 1 6 460
? 4 5
? 7
? 4 1
q 2 37 225
? 4 4
? 4 2
q 2 44 69
q 3 37 4
q 1 23 146
q 1 30 158
? 7
q 1 2 498
q 1 30 242
q 3 30 1
q 3 31 4
q 2 31 93
q 2 31 31
? 4 6
q 3 37 4
q 1 10 300
q 2 44 31
q 1 24 58
q 1 43 478
q 3 6 325
? 4 9
? 4 5
q 2 37 6
q 3 47 16
q 1 17 424
? 4 7
q 3 17 210
q 1 40 155
q 3 31 19
q 3 30 229
q 2 47 36
? 4 6
q 3 37 5
q 1 30 421
? 4 7
q 1 28 140
? 4 14
? 4 14
q 3 6 8
? 4 12
? 4 13
q 2 47 16
? 4 8
q 2 37 1
q 3 43 109
? 7
q 2 31 5
q 1 21 464
q 1 14 272
q 1 46 310
? 4 2
q 1 35 394
? 6
q 3 35 26
q 3 23 48
? 6
q 1 17 59
q 1 18 464
? 4 5
? 4 17
q 1 14 417
? 6
q 2 17 9
q 1 50 278
? 6
? 4 16
? 4 4
? 7
? 4 10
? 6
? 7
? 4 13
? 4 7
? 4 18
? 4 1
? 7
? 4 4
? 6
? 4 19
? 4 15
? 4 15
? 4 17
? 4 6
? 4 1
? 4 5