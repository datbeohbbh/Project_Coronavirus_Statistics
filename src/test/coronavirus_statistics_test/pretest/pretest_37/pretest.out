Query ? 4 1 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 148
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 108
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 148
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 517
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 1 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 108
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 239
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 108
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 37
Quantity: 4 people

Query ? 4 6 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 399
-> Has recovered : 0
-> Was dead : 1

Query ? 4 9 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 399
-> Has recovered : 0
-> Was dead : 1

Query ? 4 5 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 111
-> Has recovered : 124
-> Was dead : 4

Query ? 4 7 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 192
-> Has recovered : 0
-> Was dead : 325

Query ? 4 6 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 146
-> Has recovered : 0
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 155
-> Has recovered : 0
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 591
-> Has recovered : 0
-> Was dead : 230

Query ? 4 14 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 591
-> Has recovered : 0
-> Was dead : 230

Query ? 4 12 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 478
-> Has recovered : 0
-> Was dead : 0

Query ? 4 13 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 498
-> Has recovered : 0
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 155
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 333 people

Query ? 4 2 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 48
-> Has recovered : 100
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 31
Quantity: 129 people

Query ? 6 :
The city with the highest recovery is city with the index 31
Quantity: 129 people

Query ? 4 5 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 98
-> Has recovered : 0
-> Was dead : 48

Query ? 4 17 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 498
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 31
Quantity: 129 people

Query ? 6 :
The city with the highest recovery is city with the index 31
Quantity: 129 people

Query ? 4 16 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 464
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 87
-> Has recovered : 129
-> Was dead : 23

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 333 people

Query ? 4 10 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 278
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 31
Quantity: 129 people

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 333 people

Query ? 4 13 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 368
-> Has recovered : 0
-> Was dead : 26

Query ? 4 7 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 155
-> Has recovered : 0
-> Was dead : 0

Query ? 4 18 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 591
-> Has recovered : 0
-> Was dead : 230

Query ? 4 1 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 40
-> Has recovered : 52
-> Was dead : 16

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 333 people

Query ? 4 4 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 87
-> Has recovered : 129
-> Was dead : 23

Query ? 6 :
The city with the highest recovery is city with the index 31
Quantity: 129 people

Query ? 4 19 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 689
-> Has recovered : 0
-> Was dead : 0

Query ? 4 15 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 464
-> Has recovered : 0
-> Was dead : 0

Query ? 4 15 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 464
-> Has recovered : 0
-> Was dead : 0

Query ? 4 17 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 498
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 140
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 40
-> Has recovered : 52
-> Was dead : 16

Query ? 4 5 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 98
-> Has recovered : 0
-> Was dead : 48

